tests/stavenote_tests.js

```diff
@@ -373,7 +373,7 @@ VF.Test.StaveNote = (function() {
       tickContext.setX(10);
       tickContext.setPadding(0);
 
-      equal(tickContext.getWidth(), 16);
+      equal(tickContext.getWidth(), 15.5988);
```

Comment:
This _might_ cause platform-specific errors due to floating point variations. Does qunit have an `almostEqual`? (or perhaps define `amostEqual(value, expectedValue, errorMargin)`).